### PR TITLE
docs: document how this service is deployed

### PR DESCRIPTION
**Pull request by lucab**

### CODE REVIEW

**Comment by lucab** [comment]
This depends on https://github.com/coreos-inc/infra-kubernetes/pull/341.

File: README.md [hunk 1]
```diff
@@ -14,3 +14,14 @@ The following glide commands can be used to update the dependencies of this proj
...
+## Deployment
+
+The `master` branch of this repository is configured for autobuilding on Quay.
+The resulting image is pushed to `quay.io/coreosinc/coreos-userdata-validator:latest`,
+which is not available for pulling by the general audience.
+
+The `latest` tag of this image is deployed on CoreOS kubernetes cluster
+via the relevant Helm [chart], with an appropriate pull-secret.
+
+[chart]: https://github.com/coreos-inc/infra-kubernetes/tree/master/charts/userdata-validator
```
Comment (crawford) [inline README.md]: We probably shouldn't link to a private repo from this public one.

Comment (lucab) [reply README.md]: Indeed, removed.

**Review by crawford — APPROVED** [review]

**Comment by lucab** [comment]
Amended this to mention explicit tagging and bumping helm chart. I tagged `20180601-01`, and it has already been deployed. This is ready to go too.

**Pull request merged**